pkg/ttnpb/application.pb.paths.fm.go

```diff
@@ -0,0 +1,146 @@
+// Code generated by protoc-gen-fieldmask. DO NOT EDIT.
+
+package ttnpb
+
+var ApplicationFieldPathsNested = []string{
+	"attributes",
+	"contact_info",
+	"created_at",
+	"description",
+	"ids",
+	"ids.application_id",
+	"name",
+	"updated_at",
+}
+
+var ApplicationFieldPathsTopLevel = []string{
+	"attributes",
+	"contact_info",
+	"created_at",
+	"description",
+	"ids",
+	"name",
+	"updated_at",
+}
+var ApplicationsFieldPathsNested = []string{
+	"applications",
+}
+
+var ApplicationsFieldPathsTopLevel = []string{
+	"applications",
+}
+var GetApplicationRequestFieldPathsNested = []string{
+	"application_ids",
+	"application_ids.application_id",
+	"field_mask",
+}
+
+var GetApplicationRequestFieldPathsTopLevel = []string{
+	"application_ids",
+	"field_mask",
+}
+var ListApplicationsRequestFieldPathsNested = []string{
```

Comment:
Inconsistent spacing, but that can be fixed in the future.

Reply:
This comes from https://github.com/lyft/protoc-gen-validate please open an issue there.
I don't think we should waste time making sure the newlines in our generated code are consistent

Reply:
https://github.com/TheThingsIndustries/protoc-gen-fieldmask/issues/27